### PR TITLE
added color data

### PR DESCRIPTION
**Pull request by decobots**

### CODE REVIEW

File: src/svgmaker.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+def create_svg(color_groups: List[List['Color']]):
+    with open(os.path.join(DATA_DEFAULT_PATH, "test.svg"), mode='w') as file:
+        dwg = svgwrite.Drawing(size=(f'{W * len(color_groups[0])}px',
+                                     f'{H * len(color_groups)}px'
+                                     ))
+        for group, y in zip(color_groups, range(0, len(color_groups) * H, H)):
+            for x, color in enumerate(group):
+                rect = dwg.rect(insert=(x * W, y), size=(W, H), fill=color.hex)
+                dwg.add(rect)
+                text = dwg.text(list(map(int, color.lab_values)),
```
Comment (livercat) [inline src/svgmaker.py]: list(map(int, color.lab_values)) -> [int(v) for v in color.lab_values]

File: src/svgmaker.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+from src.color_table import Color, ColorTable
+
+DATA_DEFAULT_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data')
+W = 140
+H = 40
+
+
+def create_svg(color_groups: List[List['Color']]):
+    with open(os.path.join(DATA_DEFAULT_PATH, "test.svg"), mode='w') as file:
```
[the remark this answers is not in the document]
Comment (decobots) [reply src/svgmaker.py]: done

File: src/svgmaker.py (hunk 1 above)
Comment (decobots) [reply src/svgmaker.py]: done

File: src/svgmaker.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        dwg.write(file)
+
+
+def generate_colors(number_of_colors: int = 10, number_of_additional_colors: int = 1):
+    tab = ColorTable()
+    lst = []
+    for n in range(number_of_colors):
+        random_color = Color(r=random.randint(0, 255), g=random.randint(0, 255), b=random.randint(0, 255))
+        near = tab.nearest_color(random_color.lab).color
+        delta = tab.calculate_deltas(random_color.lab)
```
[the remark this answers is not in the document]
Comment (decobots) [reply src/svgmaker.py]: As we discussed, I plan to change this method in future

**Pull request merged**